src/views/partials/breaches.js

```diff
@@ -17,7 +17,7 @@ function createEmailCTA (count) {
 
   if (count >= total) return '' // don't show CTA if additional emails are not available for monitor
 
-  return `<button class='anchor' data-dialog='add-email'>${getMessage('add-email-link')}</button>`
+  return `<a href='dialog/add-email'>${getMessage('add-email-link')}</a>`
```

Comment:
Question (non-blocking): From the perspective of a user the behavior is more like a button. Should we add the attribute `role="button"`?

Reply:
Fair point.  I changed these to buttons in https://github.com/mozilla/blurts-server/pull/2830/commits/9efba5d249d99479124743a60d6ea6001fa48e17